### ISSUE TITLE
Keyboard shortcut to toggle visibility of active layer

### ISSUE THREAD

**Issue by incoheart** [drawpile/Drawpile#1366]
I would like a keyboard shortcut to toggle the checkbox for visibility on the active / selected layer, instead of using a mouse.  Currently the Layers dock and the Layer Properties dialogue are the only way to toggle layer visiblity.

![hidden_layer](https://github.com/user-attachments/assets/e919420b-9758-4e39-8a71-b92e0a368d90)![layer_properties](https://github.com/user-attachments/assets/32e9d28f-111d-4540-98ab-b0ed0a59ae76)

This is helpful for comparing layers while sketching. I use a shortcut like this  all the time in Krita or Paint Tool SAI 2. I think it should be called "Toggle Layer Visibility". (I usually have it configured to 1 since I use it so often.)

For semantic reasons, here is how it looks in the settings for Paint Tool Sai 2 "Change Visibility of Layer"
![sai2_change_visibility_of_layer](https://github.com/user-attachments/assets/a3970666-77b5-4cb7-8af3-853ab16c30e3)

Here is how it looks in the settings for Krita as "Toggle Layer Visibility"
![krita_toggle_layer_visibility](https://github.com/user-attachments/assets/a99e4b57-a2f8-4dba-91b9-ad0e3dd5a586)

**Comment by askmeaboutlo0m** [comment]
Yup, this makes obvious sense to have and is trivial to implement, so I did. It's available in the continuous release: <https://github.com/drawpile/Drawpile/releases/tag/continuous>

The action is called `Toggle Layer Visibility`. There's no shortcut assigned to it by default, but you can give it one via Edit → Preferences, under the Shortcuts tab. By default, <kbd>1</kbd> is assigned to `Brush slot #1`, so you'll have to clear or reassign that one to avoid a conflict.

Thanks for suggesting it!
